samples/algorithms/integer-factorization/Program.cs

```diff
@@ -126,5 +139,20 @@ static int Estimate(EstimateOptions options)
 
             return 0;
         }
+
+        static int Visualize(VisualizeOptions options) {
```

Comment:
It would be nice to update the README file and explain how the visualize command can be utilized. For example, provide a link to the flame graph tool or an example how the flame graph tool can be invoked to produce an SVG file.